chpl/chpl-resources/src/main/resources/email.properties

```diff
@@ -27,31 +27,31 @@ questionableActivityNoDataEmailBody=<p>No questionable activity was found betwee
 #pending change request email properties
 pendingChangeRequestReportFilename=pending-change-request
 pendingChangeRequestEmailSubject=Pending Change Request Report
-pendingChangeRequestHasDataEmailBody=<p>A summary of pending change requests is attached.</p><p><ul><li>%s pending Change Website Requests were found</li></ul></p>
-pendingChangeRequestNoDataEmailBody=<p>No pending change requests were found.</p>
+pendingChangeRequestHasDataEmailBody=<p>A summary of pending change requests is attached.</p>Report contains data for the following ONC-ACBs:%s<p><ul><li>%s pending Change Website Requests were found</li></ul></p>
```

Comment:
This added text ought to be surrounded by `<p>` tags to make it semantically meaningful. Similar comments for the other changes

Reply:
done